features/step_definitions/inbox_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^I am on the welcome page$/) do
+  visit welcome_index_path
+end
+
+Then(/^I am on the inbox page$/) do
+  visit mailbox_inbox_path
+end
+
+Then(/^I should see "([^"]*)" count "([^"]*)"$/) do |name1, counter|
+  @user = User.find_by(name: name1)
+  expect(@user.unread_inbox_count).to eq counter.to_i
+end
```

Comment:
so what I was looking for here is to actually SEE on the browser page the message count. This directly tests the helper method. You should be able to do `I should see "Message 1"` or something like that - somewhere it is putting the actual number on the page. That's the test I want to see.